geopandas/tests/test_array.py

```diff
@@ -192,17 +191,10 @@ def test_to_wkb():
 @pytest.mark.parametrize("string_type", ["str", "bytes"])
 def test_from_wkt(string_type):
     if string_type == "str":
-        f = six.text_type
-    else:
-        if six.PY3:
-
-            def f(x):
-                return bytes(x, "utf8")
-
-        else:
+        f = str
 
-            def f(x):
-                return x
+    def f(x):
```

Comment:
Should this still be in an `else:` block? (Because now the `f = str` above is always overriden by this def)